sdk/python/packages/flet-core/src/flet_core/navigation_bar.py

```diff
@@ -77,57 +74,63 @@ def _get_children(self):
         return children
 
     # icon
+
     @property
     def icon(self) -> Optional[str]:
         return self._get_attr("icon")
 
     @icon.setter
-    def icon(self, value: Optional[str]):
+    def icon(self, value: str) -> None:
```

Comment:
- `Optional` is actually needed here because `icon=None` is also accepted. 
Your change implies "only strings are accepted", which is not that right.

- Concerning `-> None`, we could leave it out, as python uses it by default. 
![xx](https://github.com/flet-dev/flet/assets/98978078/d7b2dcf1-58c5-48cb-b56c-65c6eb05a7d0)

Reply:
thanks for feedback